### ISSUE TITLE
module error！！！！！

### ISSUE THREAD

**Issue by daqingchong0809** [ethereum/remix-ide#2842]
If you use 'npm start' to start the package get  module error,please use 'npm run build && npm run serve' to start the package!!!!!!!!Remember, these two commands are together！！！！！！！！！！！！！！

**Comment by ryestew** [comment]
Yes - we should update the readme & replace `npm start` with `npm run build && npm run serve`

**Comment by ryestew** [comment]
Of course with the monorepo this becomes irrelevant.
